src/python/pants/backend/python/target_types.py

```diff
@@ -116,8 +116,8 @@ class PexEntryPointField(StringField, AsyncFieldMixin, SecondaryOwnerMixin):
     help = (
         "The entry point for the binary, i.e. what gets run when executing `./my_binary.pex`.\n\n"
         "You can specify a full module like 'path.to.module' and 'path.to.module:func', or use a "
-        "shorthand to specify a file name, using the same syntax as the `sources` field:\n\n    1) "
-        "'app.py', Pants will convert into the module `path.to.app`;\n    2) 'app.py:func', Pants "
+        "shorthand to specify a file name, using the same syntax as the `sources` field:\n\n  1) "
+        "'app.py', Pants will convert into the module `path.to.app`;\n  2) 'app.py:func', Pants "
```

Comment:
The fewer spaces are important so that Readme.com doesn't interpret this as a code block.